### PR TITLE
Remove unused emits.

### LINKED ISSUE #1733: Remove unused emits
Some of the components have legacy \ irrelevant `events` listed in our docs and not used anymore in the component itself.

For example `VaFileUpload` has `input` `Event` described in the documentation but we don't have this `emit` in the component:

unusedEmitsExample

- [x] Find similar cases and remove them

### PR DESCRIPTION
**Pull request by xiongmao86**
## Description
Some of the events are not emitted, and not used any more. This pr try to remove them from docs.
Fixes #1733

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/page-configs/ui-elements/button/api-options.ts [hunk 1]
```diff
@@ -4,9 +4,6 @@ export default defineManualApi({
   slots: {
     default: { },
   },
-  events: {
-    click: { types: '`() => Event`' },
```
Comment (xiongmao86) [inline packages/docs/src/page-configs/ui-elements/button/api-options.ts:8]: `VaButton` doesn't emit click, I think it is a common need, is this a bug of `VaButton`, or am I making mistakes with this sort of events?

File: packages/docs/src/page-configs/ui-elements/backtop/api-options.ts [hunk 2]
```diff
@@ -1,11 +1,6 @@
 import { defineManualApi } from '../../../components/DocsApi/ManualApiOptions'
 
 export default defineManualApi({
-  events: {
-    click: {
-      types: 'Event',
```
Comment (xiongmao86) [inline packages/docs/src/page-configs/ui-elements/backtop/api-options.ts:6]: `VaBacktop` has use this event. So there are some event didn't emitted, but are used in demo. What kind of event have this characteristic, currently I am looking at relevant files such as `VaBacktop.vue` and `VaBacktop.demo.vue`, but what if event don't have example in `*.demo.vue`?

File: packages/docs/src/page-configs/ui-elements/button/api-options.ts (hunk 1 above)
Comment (m0ksem) [reply packages/docs/src/page-configs/ui-elements/button/api-options.ts:8]: VaButton doesn't emit "click" as custom component event, its true. 
When user type:
```vue
<VaButton @click="doSmth" />
```
he actually listens to DOM event, which propagate from `button` tag.

So hypothetical user can make this code which work okay.
```vue
<VaButton @dblclick="doSmth" @contextmenu="doSmth2" />
```

So there is no bug, you can click on any vuestic component.

So, this is a lie. We do not emit any clicks 😁
image

I think we can safety remove it from docs. 
Or we can add DOM events section under spoiler, so no one will be confused with missing events. And add all DOM Mouse events: https://developer.mozilla.org/en-US/docs/Web/Events
What do you think?

File: packages/docs/src/page-configs/ui-elements/backtop/api-options.ts (hunk 2 above)
Comment (m0ksem) [reply packages/docs/src/page-configs/ui-elements/backtop/api-options.ts:6]: Same. It is DOM event, not vue emit.

**Review by m0ksem — APPROVED** [review]

File: packages/docs/src/page-configs/ui-elements/button/api-options.ts (hunk 1 above)
Comment (m0ksem) [reply packages/docs/src/page-configs/ui-elements/button/api-options.ts:8]: /cc @asvae

Comment (xiongmao86) [reply packages/docs/src/page-configs/ui-elements/button/api-options.ts:8]: Thanks, @m0ksem, I'll wait for your discussion.

Comment (Derranion) [reply packages/docs/src/page-configs/ui-elements/button/api-options.ts:8]: Yes, it will be very confusing for the users to remove this event from docs.

**Review by Derranion — APPROVED** [review]
Thanks a lot for your help.

File: packages/docs/src/page-configs/ui-elements/button/api-options.ts (hunk 1 above)
Comment (Derranion) [reply packages/docs/src/page-configs/ui-elements/button/api-options.ts:8]: Hey @xiongmao86 

We will handle it in the other task to not bother you :)

Would be amazing if you could just add to such events description in the docs something like this in the end.

For example for this click `Event` description will be:

```
Emitted when user clicks on button
(native DOM event)
```

Thanks again 👍

Comment (xiongmao86) [reply packages/docs/src/page-configs/ui-elements/button/api-options.ts:8]: Thanks, @Derranion. I'll do what you said.

**Comment by xiongmao86** [comment]
This pr is ready for review.

**Comment by xiongmao86** [comment]
Thanks, @m0ksem and @Derranion.

**Pull request merged**